### PR TITLE
Part of book/report

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

**Review by brinxmat — APPROVED** [review]

File: docs/cristin.ttl [hunk 1]
```diff
@@ -228,8 +247,10 @@
 :FAGFELT_Class
     a               rdfs:Class ;
     rdfs:subClassOf :CristinClass ;
-    rdfs:comment    """The scientific field in which the book is published""" ;
-    rdfs:subClassOf :TYPE_BOK_RAPPORT_Class .
+    rdfs:comment    """The scientific field in which the book/chapter is published
+    When the domain is TYPE_BOK_RAPPORT_DEL the respective field is always null.
+    """ ;
+    rdfs:domain :TYPE_BOK_RAPPORT_Class, :TYPE_BOK_RAPPORT_DEL_Class  .
```
Comment (kariht) [inline docs/cristin.ttl:253]: vet ikke helt om jeg forstod dette, men det er eksempler hvor både bok_rapport og tilhørende del har fagfelt utfylt i begge og nødvendigvis ikke sammekode heller

File: docs/cristin.ttl [hunk 2]
```diff
@@ -151,11 +151,24 @@
     Owing to the number of funding bodies, there is no uniform way to dereference grants externally.
     """ .
 
-:TYPE_BOK_RAPPORT_Class
-    a               :CristinClass ;
-    rdfs:comment    "Indicates that the respective PRESENTASJON_VARBEID_Class instance is a reference to a Cristin project" ;
-    rdfs:domain     :Result ;
-    rdfs:subClassOf :PRESENTASJONSTYPEKODE_VALUES_Class .
```
Comment (Antares42) [inline docs/cristin.ttl:158]: This definition is in the document twice, and this one here is faulty, so I'm removing it.

File: docs/cristin.ttl [hunk 3]
```diff
@@ -915,10 +926,12 @@ It is a self reference.""" ;
 :antall_sider_totalt
     a               :cristinfield ;
     rdfs:comment    """
-    The total number of pages.
+    The total number of pages of the specific entry.
+    When the the entry represents a part of another entry (e.g. a Chapter in a Book) then the total number of pages
+    refers to the entry (e.g Chapter) and not the to parent entry (e.g. Book).
```
Comment (Antares42) [inline docs/cristin.ttl:931]: Looks like we're now using a bunch of terms for what one cristin result id represents: "result", "entry", "entity", "publication"...? Do we need to consolidate?

File: docs/cristin.ttl [hunk 4]
```diff
@@ -1243,12 +1247,65 @@ It is a self reference.""" ;
...
+
+:varbeidlopenr_inngar_i
+    a            :cristinfield ;
+    rdfs:comment """The CristinID of the current TYPE_BOK_RAPPORT_DEL_Class entry's parent TYPE_BOK_RAPPORT_Class entry.""" ;
+    rdfs:domain  :TYPE_BOK_RAPPORT_DEL_Class ;
+    rdfs:range   xsd:integer .
+
+:status_utgitt_av_forlag
+    a            :cristinfield ;
+    rdfs:comment """UNKNOWN meaning. It takes values "J" and "N" corresponding to the values true and false.""" ;
```
Comment (Antares42) [inline docs/cristin.ttl:1273]: We are investigating what the meaning of that field is in this context. Will update accordingly.

File: docs/cristin.ttl (hunk 3 above)
Comment (axthosarouris) [reply docs/cristin.ttl:931]: I will create a separate PR for this

**Pull request merged**